src/APIFunctions/3DPrinting.js

```diff
@@ -15,7 +23,7 @@ import axios from 'axios'
  * @returns {boolean} - If the request was successfully submitted or not.
```

Comment:
Be sure to update the documentation of this comment, we don't return a boolean anymore

```suggestion
 * @returns {ApiResponse} Containing any error information related to the 
 * request
```